scst-scan/integrate-app-scanning.hbs.md

```diff
@@ -0,0 +1,113 @@
+# Integration Guide
+
+This topic describes how to integrate Supply Chain Security Tools - Scan 2.0 with the Out of the Box Supply Chain.
+
+## <a id="prerequisites"></a> Prerequisites
+
+Before you can integrate Supply Chain Security Tools - Scan 2.0 with the Out of the Box Supply Chain:
+- Create an ImageVulnerabilityScan. You can either integrate your scanner or select from the provided samples:
```

Comment:
```suggestion
- Select an ImageVulnerabilityScan. You can either bring your own scanner by creating a custom ImageVulnerabilityScan template or select from the provided samples:
```

I think selecting an option is more appropriate than create right?  Since if you are using one of the sample templates you don't need to create it?